sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -1040,15 +1047,18 @@ public void captureLog(
             .recordLostEvent(DiscardReason.BEFORE_SEND, DataCategory.LogItem);
         return;
       }
+
+      loggerBatchProcessor.add(logEvent);
     }
 
-    try {
-      final @NotNull SentryEnvelope envelope =
-          buildEnvelope(new SentryLogEvents(Arrays.asList(logEvent)), traceContext);
+    hint.clear();
+  }
 
-      hint.clear();
-      // TODO buffer
-      sendEnvelope(envelope, hint);
+  @Override
+  public void captureBatchedLogEvents(final @NotNull SentryLogEvents logEvents) {
```

Comment:
shouldn't this be Internal?
The user shouldn't call it directly, right?

Reply:
Marked internal in https://github.com/getsentry/sentry-java/pull/4392